From: Victor Seva <linuxmaniac@torreviejawireless.org>
Date: Wed, 23 Sep 2020 09:01:43 +0200
Subject: [PATCH] ndb_redis: set message level to debug on exec

* logging error makes no sense here since there's going
  to be a reconnection afterwards

> ERROR: ndb_redis [redis_client.c:903]: redisc_exec_argv(): Redis error: Server closed the connection

(cherry picked from commit a9c331d70dbed6827874aa99296203ac5a9d5c7b)
---
 src/modules/ndb_redis/redis_client.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/ndb_redis/redis_client.c b/src/modules/ndb_redis/redis_client.c
index 5ad063f..bd97012 100644
--- a/src/modules/ndb_redis/redis_client.c
+++ b/src/modules/ndb_redis/redis_client.c
@@ -770,7 +770,7 @@ int redisc_exec_pipelined(redisc_server_t *rsrv)
 		/* null reply, reconnect and try again */
 		if (rsrv->ctxRedis->err)
 		{
-			LM_ERR("Redis error: %s\n", rsrv->ctxRedis->errstr);
+			LM_DBG("Redis error: %s\n", rsrv->ctxRedis->errstr);
 		}
 		if (redisc_create_pipelined_message(rsrv) == 0)
 		{
